### PR TITLE
Added German Readme

### PR DESCRIPTION
**Pull request by DontFred**

### CODE REVIEW

**Comment by DontFred** [comment]
Maybe you could change the documentation to an open source project as well.  Then we could help you to maintain it. Possible is to use [nextra](https://nextra.site/) with markdown.

File: README-de.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+Dokploy ist eine kostenlose und self-hostable Platform as a Service (PaaS), welche das hosten und managen von deinen Projekten und Datenbanken vereinfacht, das geschieht mithilfe von Docker und Treafik. Es ist designt, um deine Leistung und die Sicherheit deiner Projekte zu verbessern. Dokploy erlaubt dir schnell und einfach auf jeder VPS deine Projekte zu verwirklichen.
+
+
+## Explanation
+[English](README.md) | [中文](README-zh.md) | [Deutsch](README-de.md)
+
+
+
+
+## 🌟 Features
```
Comment (Siumauricio) [inline README-de.md:22]: Change to a german language

File: README-de.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+<div align="center" style="width:100%;">
+<img src="https://raw.githubusercontent.com/Dokploy/dokploy/main/logo.png" alt="Reflex Logo"  style="width:60%;">
+</div>
+<hr>
+
+
+Dokploy ist eine kostenlose und self-hostable Platform as a Service (PaaS), welche das hosten und managen von deinen Projekten und Datenbanken vereinfacht, das geschieht mithilfe von Docker und Treafik. Es ist designt, um deine Leistung und die Sicherheit deiner Projekte zu verbessern. Dokploy erlaubt dir schnell und einfach auf jeder VPS deine Projekte zu verwirklichen.
+
+
+## Explanation
```
Comment (Siumauricio) [inline README-de.md]: Change to a german language

File: README-de.md (hunk 1 above)
Comment (Siumauricio) [inline README-de.md]: Change to a german language

File: README-de.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+```bash
+curl -sSL https://dokploy.com/install.sh | sh
+```
+
+Getestete Systems:
+
+- Ubuntu 20.04
+- Debian 11
+
+## 📄 Documentation
```
Comment (Siumauricio) [inline README-de.md]: Change to a german language

File: README-de.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+- **Projekte**: - **Projekte**: Hoste jegliche Art von Projekt (Node.js, PHP, Python, Go, Ruby, etc.) mit Einfachheit.
+- **Datenbanken**: Erstelle und manage Datenbanken, wie MySQL, PostgreSQL, MongoDB, MariaDB, Redis, und mehr.
+- **Docker Management**: Einfach Docker container hosten und managen.
+- **Traefik Integration**: Automatische Integration mit Traefik für routing und load balancing
+- **Real-time Monitoring**: Monitor von CPU, RAM, Speicher, und network Nutzung.
+- **Database Backups**: Automatische Backups mit Support für mehrere Speicher Systeme.
+
+
+## 🚀 Getting Started
```
Comment (Siumauricio) [inline README-de.md]: Change to a german language

**Review by Siumauricio — CHANGES_REQUESTED** [review]
please update the titles to a german language

**Review by Siumauricio — APPROVED** [review]
Thanks!

**Pull request merged**